pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -75,6 +75,25 @@ private JsonFunctions() {
     CacheProvider.setCache(new JsonPathCache());
   }
 
+  @ScalarFunction
+  public static String jsonExtractScalar(String jsonString, String jsonPath) {
```

Comment:
The signature of this function should be using the same arguments as the transform function definition:
```
JSONEXTRACTSCALAR(jsonField, 'jsonPath', 'resultsType', [defaultValue])
```

You can refer to this doc(https://docs.pinot.apache.org/functions-1/jsonextractscalar
) and the implementation of `JsonExtractScalarTransformFunction`.